src/SDKs/IotHub/Management.IotHub/generate.cmd

```diff
@@ -4,4 +4,4 @@
 ::
 
 @echo off
-call %~dp0..\..\..\..\tools\generate.cmd iothub/resource-manager %*
+call %~dp0..\..\..\..\tools\generate.cmd provisioningservices/resource-manager %*
```

Comment:
@montgomp why iotHub has been changed to provisioning services?

Reply:
Thanks for the catch.